### PR TITLE
Bug 1837323: Fixed EO crash

### PR DESCRIPTION
**Pull request by igor-karpukhin**
* Added interface type check

### CODE REVIEW

File: pkg/elasticsearch/indices.go [hunk 1]
```diff
@@ -95,9 +96,19 @@ func (ec *esClient) AddAliasForOldIndices() bool {
 		}
 
 		if payload.ResponseBody[index] != nil {
-			indexBody := payload.ResponseBody[index].(map[string]interface{})
+			indexBody, ok := payload.ResponseBody[index].(map[string]interface{})
+			if !ok {
+				logger.Warnf("unable to unmarshal index response body. Type: %s",
```
Comment (jcantrill) [inline pkg/elasticsearch/indices.go:101]: Please add more info as whats going on here and to what. something like: 

`logger.Warnf("unable to unmarshal response body of cluster %q index %q to add alias . Type: %s",...."`

same with the other message

Comment (igor-karpukhin) [reply pkg/elasticsearch/indices.go]: Fixed

File: pkg/elasticsearch/indices.go [hunk 2]
```diff
@@ -95,9 +96,23 @@ func (ec *esClient) AddAliasForOldIndices() bool {
 		}
 
 		if payload.ResponseBody[index] != nil {
-			indexBody := payload.ResponseBody[index].(map[string]interface{})
+			indexBody, ok := payload.ResponseBody[index].(map[string]interface{})
+			if !ok {
+				logger.Warnf("unable to unmarshal index '%s' response body for cluster '%s'. Type: %s",
+					index,
+					ec.cluster,
+					reflect.TypeOf(payload.ResponseBody[index]).String())
```
Comment (alanconway) [inline pkg/elasticsearch/indices.go:104]: You can print the type more succinctly (same result) with the %T format directive like this:
```
logger.Warnf("unable to unmarshal index '%s' response body for cluster '%s'. Type: %T", index, ec.cluster,
ec.cluster)
```

**Pull request merged**